leftwm-core/src/handlers/window_handler.rs

```diff
@@ -439,10 +439,10 @@ pub fn scratchpad_xyhw(xyhw: &Xyhw, scratch_pad: &ScratchPad) -> Xyhw {
 fn sane_dimension(config_value: Option<Size>, default_percent: f32, max_pixel: i32) -> i32 {
     match config_value {
         Some(size) => match size {
-            Size::Percentage(percentage) if (0.0..0.9).contains(&percentage) => {
+            Size::Percentage(percentage) if (0.0..=1.0).contains(&percentage) => {
```

Comment:
Does this:
```suggestion
            Size::Percentage(percentage) @ 0.0..=1.0 => {
```
or this:
```suggestion
            Size::Percentage(percentage @ 0.0..=1.0) => {
```
Work?

Reply:
Wow I didn't know that syntax could be used like that. Really cool!!

Unfortunately it doesn't seem to work because of the floats... it would have work for integers though!

```
warning: floating-point types cannot be used in patterns
   --> leftwm-core/src/handlers/window_handler.rs:442:36
    |
442 |             Size::Percentage(0.0..=1.0) => {
    |                                    ^^^
    |
    = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
    = note: for more information, see issue #41620 <https://github.com/rust-lang/rust/issues/41620>
```

The one for Pixel doesn't work because there is a variable inside:

```
error[E0080]: runtime values cannot be referenced in patterns
   --> leftwm-core/src/handlers/window_handler.rs:445:37
    |
445 |             Size::Pixel(pixel @ 0..=max_pixel) => pixel,
    |  
```

I would gladly replace by `if pixel <= max_pixel` but this won't work either because max_pixel is i32.